### DISCUSSION TITLE
Weird behavior when a Barbarian hero visits a Barbarian castle Month 4 Week 1 Day 1, and this results in strange captured artifacts

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#7634] [General]
In fheroes2, there is a weird behavior when a Barbarian hero visits a Barbarian castle Month 4 Week 1 Day 1, and this results in strange captured artifacts.

In fheroes2:
https://youtu.be/T5Z8wAmx_Fg

**Comment by Branikolog** [comment]
@LeHerosInconnu 
Isn't it the expansion artifacts?

**Comment by LeHerosInconnu** [comment]
Hello @Branikolog,

> @LeHerosInconnu
> Isn't it the expansion artifacts?

In some way.
The last one is particularly devastating. :D

**Comment by ihhub** [comment]
I am moving this bug even against our policy for the discussion section :)
